libs/slam/async_slam/async_slam_worker.cpp

```
@@ -0,0 +1,282 @@
+      std::lock_guard image_guard(processing_images_mutex_);
+      current_images = processing_images_;
+    }
+    const auto current_image =
+        std::find_if(current_images.begin(), current_images.end(), [](const auto& image) { return image != nullptr; });
+    const bool is_valid_image =
+        current_image != current_images.end() && ((*current_image)->get_image_meta().frame_id == frame_data.frame_id);
+    Isometry3T pose_estimate_slam;
+    {
+      const VOTrackData& track_data = vo_kf->track_data;
+
+      Isometry3T last_keyframe_pose;
```

Clang (14.0.6)

[warning] variable 'last_keyframe_pose' is not initialized

(cppcoreguidelines-init-variables)